test/integration/apiserver/apply/apply_test.go

```diff
@@ -291,6 +291,19 @@ func TestApplyManagedFields(t *testing.T) {
 		t.Fatalf("Failed to create object using Apply patch: %v", err)
 	}
 
+	_, err = client.CoreV1().RESTClient().Patch(types.MergePatchType).
+		Namespace("default").
+		Resource("configmaps").
+		Name("test-cm").
+		Param("fieldManager", "updater").
+		Body([]byte(`{"data":{"new-key": "value"}}`)).Do().Get()
+	if err != nil {
+		t.Fatalf("Failed to patch object: %v", err)
+	}
+
+	// Sleep for one second to make sure that the times of each operation are recorded separately
+	time.Sleep(1 * time.Second)
```

Comment:
i'd add some more detail to the above comment; perhaps not as full as the PR description, but just to clarify the timeout a bit more?

Reply:
fixed